docs/worker-node/install-singularity.md

```diff
@@ -49,23 +50,27 @@ As with all OSG software installations, there are some one-time (per host) steps
 - Ensure the host has [a supported operating system](/release/supported_platforms)
 - Obtain root access to the host
 - Prepare the [required Yum repositories](/common/yum)
-- A [CVMFS installation](/worker-node/install-cvmfs) for singularity image distribution
+
+In addition, with Singularity this is highly recommended:
+
+- Install [CVMFS](/worker-node/install-cvmfs)
 
 Choosing Unprivileged vs Privileged Singularity
 -----------------------------------------------
 
-There are two separate sets of instructions on this page:
+There are two sets of instructions on this page:
 
-- [Enabling unprivileged singularity](#unprivileged-singularity) via OASIS
-- [Installing privileged singularity](#privileged-singularity) via RPM
+- [Enabling Unprivileged Singularity](#enabling-unprivileged-singularity)
+- [Singularity via RPM](#singularity-via-rpm)
 
-As of December 2018, no VO in the OSG is ready to use unprivileged, non-setuid
-singularity from OASIS in production.
+As of April 2019, no VO in the OSG is ready to run unprivileged, non-setuid
```

Comment:
This makes it seem like if the admin installs unprivileged singularity, then singularity jobs won't run on their machine... in which case, why would they bother to do that?

Reply:
It was certainly not my intention to convey that!  I will rephrase.